.pnp.cjs

```diff
@@ -40592,7 +40592,7 @@ function makeApi(runtimeState, opts) {
         if (result === false) {
           throw makeError(
             ErrorCode.BUILTIN_NODE_RESOLUTION_FAILED,
-            `The builtin node resolution algorithm was unable to resolve the requested module (it didn't go through the pnp resolver because the issuer was explicitely ignored by the regexp)
+            `The builtin node resolution algorithm was unable to resolve the requested module (it didn't go through the pnp resolver because the issuer was explicitly ignored by the regexp)
```

Comment:
L40595 makes me think this is auto-generated. Probably want to add this file to the skip list.

Reply:
I ignored that file now